generate-go.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+if [[ -z "$1" ]]
+then
+  out_path="."
+else
+  out_path=$1
+fi
+
+if [[ $out_path != .* ]] && [[ $out_path != /* ]]
+then
+  echo "Path must start with '.' or with '/'"
```

Comment:
Why it's required?

Reply:
Removed. This is requirement of `protoc` if `go_package` is not a URI.